rules.mk

```diff
@@ -276,7 +276,9 @@ BASH:=bash
 TAR:=tar
 FIND:=find
 PATCH:=patch
-PYTHON:=python
+PYTHON:=python2
```

Comment:
This seems wrong as SetupHostCommand is going to create `python` symlink, not `python2` symlink.

Reply:
I should add a python2 SetupHostCommand then